test/Microsoft.TemplateEngine.TemplateLocalizer.EndToEndTests/Microsoft.TemplateEngine.TemplateLocalizer.EndToEndTests.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>$(NETCoreTargetFramework)</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
```

Comment:
I think it's worth to mark it also as "integration test project" (IsIntegrationTestProject) so it runs after unit tests.

Reply:
I renamed the project to IntegrationTests and removed this property from the csproj file.